README.md

```diff
@@ -20,6 +20,7 @@ Required Dependencies:
 * numpy
 * scipy
 * ase
+* pymatgen
```

Comment:
can pymatgen be installed in pip? Did you test in a new conda environment?

Reply:
Pymatgen can be installed in pip. The program has been tested in a new conda environment.